# ask_map_size

---
int32 row
int32 col
